lib/Channel.js

```diff
@@ -219,11 +219,6 @@ class Channel {
             this.initialize()
         }
         return TwilioIPMessagingMessages.getLastMessages(this.sid, count)
-        .then((messages) => {
```

Comment:
yes this is needed... but I found that without explicit returns that messages aren't loaded... also this should be in the getMessages function not the sendMessage function. @bradbumbalough